display/pager.go

```diff
@@ -59,24 +57,39 @@ func isPagerNeeded(content string) bool {
 	return false
 }
 
-func winMorePager(content string) {
-	// for the time being, route this to the nullpager
-	// eventually use windows `more` with a temp file
-	nullPager(content)
-}
-
 func nullPager(content string) {
 	// just dump the whole thing out
 	// we will use this for non-tty environments as well
 	fmt.Print(content)
 }
 
-func nixLessPager(content string) {
-	lessProcess := exec.Command("less", "-SRXF")
-	lessProcess.Stdout = os.Stdout
-	lessProcess.Stderr = os.Stderr
-	lessProcess.Stdin = strings.NewReader(content)
+func nixMoreOrLessPager(content string) {
```

Comment:
```
func nixPager(content string) {
	if isLessAvailable() {
		pager(exec.Command("less", "-SRXF"), content)
	} else if isMoreAvailable() {
		pager(exec.Command("more"), content)
	} else {
		nullPager(content)
	}
}

func pager(cmd *exec.Cmd){}
	cmd.Stdout = os.Stdout
	cmd.Stderr = os.Stderr
	cmd.Stdin = strings.NewReader(content)
	// Run it, so that this blocks out the go-prompt stuff.
	// No point Start-ing it anyway
	err := cmd.Run()
	if err != nil {
		utils.ShowErrorWithMessage(err, "could not display results")
	}
}
```

Reply:
done